src/static/css/cfpb-chart-builder.less

```diff
@@ -1,4 +1,4 @@
-@import 'https://code.highcharts.com/css/highcharts.css';
+@import 'highcharts.less';
```

Comment:
Instead of creating this file how would you feel about importing the `highcharts.css` file that's in `/node_modules/highcharts`? It would ensure we're always using the correct CSS file in the event we upgrade to a newer version of highcharts. It would also keep a third-party lib out of the source directory.

Try changing the above line to `@import (inline) 'highcharts.css';` and in gulp's config file add the highcharts dir to its search paths on lines [48](https://github.com/cfpb/cfpb-chart-builder/blob/master/gulp/config.js#L48) and [57](https://github.com/cfpb/cfpb-chart-builder/blob/master/gulp/config.js#L57) by changing them to: `paths: glob.sync( loc.lib + '/cf-*/src/' ).concat( [ loc.lib + '/highcharts/css/' ] )`.

That should inline highcharts.css into our stylesheet.